api/api/v1alpha1/manifest_types.go

```diff
@@ -35,21 +35,38 @@ type CustomState struct {
 	State      string `json:"state,omitempty"`
 }
 
-// ChartInfo defines chart information
-type ChartInfo struct {
-	ChartPath    string `json:"chartPath,omitempty"`
-	RepoName     string `json:"repoName,omitempty"`
-	Url          string `json:"url,omitempty"`
-	ChartName    string `json:"chartName,omitempty"`
-	ReleaseName  string `json:"releaseName,omitempty"`
-	ClientConfig string `json:"clientConfig,omitempty"`
-	Overrides    string `json:"overrides,omitempty"`
+// InstallInfo defines installation information
+type InstallInfo struct {
+	OCIRef `json:",inline"`
+	Type   string `json:"type"`
+	Name   string `json:"name"`
+}
+
+// OCIRef defines OCI image configuration
+type OCIRef struct {
+	Repo    string `json:"repo"`
+	Module  string `json:"module"`
+	Tag     string `json:"tag,omitempty"`
+	Version string `json:"version,omitempty"`
+	Digest  string `json:"digest,omitempty"`
 }
 
 // ManifestSpec defines the specification of Manifest
 type ManifestSpec struct {
-	Charts       []ChartInfo   `json:"charts,omitempty"`
+	// OCIRef specifies OCI image configuration for Manifest
+	// +optional
+	OCIRef OCIRef `json:"ociRef,omitempty"`
```

Comment:
Oups I think we changed that by accident to ociRef, should probably be called config in the JSON still